src/fsharp/service/IncrementalBuild.fsi

```diff
@@ -70,7 +70,13 @@ type internal PartialCheckResults =
       LatestImplementationFile: TypedImplFile option 
       
       /// Represents latest inferred signature contents.
-      LatestCcuSigForFile: ModuleOrNamespaceType option}
+      LatestCcuSigForFile: ModuleOrNamespaceType option
+      
+      /// If enabled, stores a linear list of ranges and strings that identify an Item(symbol) in a file. Used for background find all references.
```

Comment:
> Used for background find all references.
	
What does "background find all references" mean in this context, is there a background thread that things are calculated on? Also does this affect finding symbols in a rename operation? What about finding symbols via ctrl+T?

Reply:
Information from IncrementalBuild is referred to as the background compiler. We have several APIs with the word "Background" in them to associate with that distinction.

Reply:
Your last two questions are not related this API in particular, it's better to look at how it's actually happening in FSharp.Editor.